mlir/lib/Dialect/AMDGPU/Transforms/EmulateAtomics.cpp

```diff
@@ -189,6 +189,16 @@ void mlir::amdgpu::populateAmdgpuEmulateAtomicsPatterns(
     } else {
       target.addIllegalOp<RawBufferAtomicFmaxOp>();
     }
+    // TODO: refactor this to avoid hardcoding ISA version:
+    // https://github.com/llvm/llvm-project/issues/129206
+    // gfx950 has bf16 atomics
```

Comment:
```suggestion
    // TODO(https://github.com/llvm/llvm-project/issues/129206): Refactor
    // this to avoid hardcoding ISA version: gfx950 has bf16 atomics.
```